src/managers/input/Tablets.cpp

```diff
@@ -160,7 +167,13 @@ void CInputManager::onTabletTip(CTablet::STipEvent e) {
     const auto PTAB  = e.tablet;
     const auto PTOOL = ensureTabletToolPresent(e.tool);
     const auto POS   = e.tip;
-    g_pPointerManager->warpAbsolute(POS, PTAB);
+
+    if (PTAB->relativeInput)
+        g_pPointerManager->move({0, 0});
+    else {
```

Comment:
no {}

Reply:
done

Reply:
there are still {} around else

Reply:
I'm so sorry for wasting your time. Not sure how that happened.